website/source/docs/post-processors/vagrant.html.md

```diff
@@ -34,6 +34,7 @@ providers.
 -   DigitalOcean
 -   Google
 -   Hyper-V
+-   LXC
```

Comment:
Is there anywhere else in this file I need to add LXC @SwampDragons ?

Reply:
Probably also `The available provider names are: aws, digitalocean, google, virtualbox, vmware, and parallels`